examples/step-40/step-40.cc

```diff
@@ -403,19 +400,18 @@ namespace Step40
                      0.5 +
                        0.25 * std::sin(4.0 * numbers::PI *
                                        fe_values.quadrature_point(q_point)[0]) ?
-                   1 :
-                   -1);
+                   1. :
+                   -1.);
 
               for (unsigned int i = 0; i < dofs_per_cell; ++i)
                 {
                   for (unsigned int j = 0; j < dofs_per_cell; ++j)
-                    cell_matrix(i, j) += (fe_values.shape_grad(i, q_point) *
-                                          fe_values.shape_grad(j, q_point) *
-                                          fe_values.JxW(q_point));
+                    cell_matrix(i, j) += fe_values.shape_grad(i, q_point) *
+                                         fe_values.shape_grad(j, q_point) *
+                                         fe_values.JxW(q_point);
 
-                  cell_rhs(i) +=
-                    (rhs_value * fe_values.shape_value(i, q_point) *
-                     fe_values.JxW(q_point));
+                  cell_rhs(i) += rhs_value * fe_values.shape_value(i, q_point) *
+                                 fe_values.JxW(q_point);
```

Comment:
Should we split this into three lines?

Reply:
We do that everywhere else so yes, lets do it here too.